ted2zim/scraper.py

```diff
@@ -433,6 +438,14 @@ def get_lang_code_from_url(self, url, with_full_query=False):
             return current_lang, query
         return current_lang
 
+    def already_visited_video(self, url):
+        """ checks if a video URL is already visited """
+
+        relative_path = urllib.parse.urlparse(url)[2]
+        if relative_path in self.already_visited:
```

Comment:
skip the var here, not that useful

Reply:
Actually removed that whole function and checked directly. Also added a comment there